compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/types/BType.java

```diff
@@ -162,6 +162,23 @@ public String getQualifiedTypeName() {
         return tsymbol.pkgID.toString() + ":" + tsymbol.name;
     }
 
+    public long getFlags() {
```

Comment:
I don't think any of the child classes needs to override this method, therefore shall we make this `final`?
```suggestion
    public final long getFlags() {
```